src/components/SubHeader/EvergreenTaskSubHeader.tsx

```diff
@@ -70,12 +71,15 @@ export const EvergreenTaskSubHeader: React.FC<Props> = ({
     patchNumber,
     status,
     versionMetadata,
+    tests,
   } = loadedTask;
   const { isPatch, projectIdentifier, message, revision } = versionMetadata;
 
-  const currentTest = logkeeperBuildMetadata?.tests?.find(
-    (test) => test.id === testID
-  );
+  const currentTest = isResmoke
```

Comment:
Is there a plan to do this for normal test results as well?

Reply:
I think there's a problem with the format for normal test results. For example, if you look at the test results for [this task](https://spruce.mongodb.com/task/evergreen_ubuntu2004_test_thirdparty_docker_patch_4b2374178b940788892dddae673a2c103cfd21ca_6447ef851e2d17c4491fa5bd_23_04_25_15_19_34/tests?execution=0&sortBy=STATUS&sortDir=ASC), all of the tests have the same name `"output.thirdparty-docker.test"` in Parsley and are only distinguished by their values for `shareLine`. So it makes it kind of makes it difficult to tell what test the user actually clicked on initially

Reply:
hello I have looked 🧐 I feel like it's technically possible since `parsleyUrl` does store the `shareLine` for each test but it's kind of handwavy since you would be matching on a number rather than a real ID. There is also question of how necessary the status is, since for Evergreen test logs the status is usually immediately visible versus resmoke test logs it's hard to tell at a glance